game/src/sandbox/dashboards/commuter.rs

```diff
@@ -182,7 +214,10 @@ impl State for CommuterPatterns {
                     self.composite.replace(
                         ctx,
                         "current",
-                        "Nothing selected".draw_text(ctx).named("current"),
+                        "None selected"
```

Comment:
Ok, this is pretty silly, but I changed the empty state from "Nothing selected" to "None
selected" to avoid an undesirable shift in the commuter view.

I think the shift is because the height of the text widget seems to vary
with the height of the characters _in_ the text widget. So the low
hanging tail of "g" was causing "Nothing" to render taller than "None".

Here are the two version rendered alongside each other - notice how things shift below the word "Nothing"
![image](https://user-images.githubusercontent.com/217057/87861762-70f03a80-c906-11ea-92dc-49b62c27e720.png)

Or is there currently a way to make the text widget maintain a consistent height?

Reply:
I think "none selected" is a fine workaround for now. Not sure how to make the line of text always have the same height; I thought that should already be true, but the line rendering code is scary old stuff I haven't touched in ages:

https://github.com/dabreegster/abstreet/blob/7ad7c7c477cbde1627651fa3f5076bcebb296b58/ezgui/src/text.rs#L274

There are probably subtle bugs there.

Reply:
blechh, there's still a bit of line height dancing when multiple building types are listed - I think due to the comma. 

Would you rather merge this as is, or hold off until that can be addressed?